kurtosis-devnet/tests/eigenda/v2_migration_test.go

```diff
@@ -0,0 +1,61 @@
+package eigenda
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/geth"
+	"github.com/stretchr/testify/require"
+)
+
+// TestEigenDAV2Migration tests a rollup migration from eigenDA V1 to V2.
+// We use the proxy's admin REST API to change the dispersal backend.
+// See https://github.com/Layr-Labs/eigenda-proxy?tab=readme-ov-file#on-the-fly-migration for details.
+// This test simply checks that the batcher txs have the correct version byte.
+// The batches inbox transactions are queried via geth's GraphQL API.
+//
+// Note: because this test modifies the proxy's state config, it should be run in isolation (sequentially).
+func TestEigenDAV2Migration_Memstore(t *testing.T) {
```

Comment:
Based on the name, this is using `Memstore`. Any reason you're not using `Holesky`?

Reply:
I didn't want holesky tests to start taking 40 mins (trying to stay around the 20 min mark). But I ugess this is only 4 mins, so I could run it on holesky as well.

Reply:
added in [35a545b](https://github.com/Layr-Labs/optimism/pull/44/commits/35a545b265113838546dc303a968959c88f2bd1a), lets see how long CI takes with this.